apps/nextjs-website/src/app/page.tsx

```diff
@@ -5,13 +5,19 @@ import RelatedLinks from '@/components/atoms/RelatedLinks/RelatedLinks';
 import News from '@/components/organisms/News/News';
 import ProductsShowcase from '@/components/organisms/ProductsShowcase/ProductsShowcase';
 import { getProducts } from '@/lib/api';
+import WebinarHeaderBanner from '@/components/atoms/WebinarHeaderBanner/WebinarHeaderBanner';
 
 const Home = async () => {
   const products = await getProducts();
   const { homepage, header } = translations;
 
   return (
     <>
+      <WebinarHeaderBanner
+        slug={products[0].slug}
+        text={'Comunicazioni a valore legale. Ti aspettiamo al webinar!'}
+      />
+
```

Comment:
Now that https://github.com/pagopa/developer-portal/pull/359 is merged webinar can be retrived from api (once that branch is allign to main).
```suggestion
      {nextWebinars.length !== 0 && (<WebinarHeaderBanner
        slug={nextWebinars[0].path}
        text={nextWebinars[0].title}
      />
      })

```